### PR TITLE
chore(TimePicker-compat): add VR test

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
Add VR test.
Since TimePicker is built on Combobox, the VR test focuses on TimePicker specifics and skips the tests like appearances/sizes that are tested in Combobox

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3129d7868082c057cc01ad860a45a9ffc8072a7b'>3129d7868082c057cc01ad860a45a9ffc8072a7b</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/vr-tests-react-components/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "@fluentui/react-text": "*",
```
Comment (fabricteam) [inline apps/vr-tests-react-components/package.json:65]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29732&commitId=779e82bd1093488a530c92855f1107fa4f3987a5&env=prod&clientType=FLUENTUI)


  <details><summary><b>TimePicker compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TimePicker compat.Default.default.chromium.png|0| Added|
|TimePicker compat.Show seconds.default.chromium.png|0| Added|
|TimePicker compat.With 12-hour format.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: apps/vr-tests-react-components/src/stories/TimePicker.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { TimePicker } from '@fluentui/react-timepicker-compat-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('TimePicker compat', module)
```
Comment (Hotell) [inline apps/vr-tests-react-components/src/stories/TimePicker.stories.tsx:7]: curious why do we allow to use long deprecated story format  ?  I see mix/match of this within our VR test stories. 

@ling1726 @TristanWatanabe  

lets rewrite this to CSF format

Comment (ling1726) [reply apps/vr-tests-react-components/src/stories/TimePicker.stories.tsx:7]: Agreed, new vr tests should be implemented with CSF. You can use the following as an example

* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx
* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Toolbar/Toolbar.stories.tsx
* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Toast/Toast.stories.tsx

Comment (YuanboXue-Amber) [reply apps/vr-tests-react-components/src/stories/TimePicker.stories.tsx:7]: Updated :)

**Review by ling1726 — APPROVED** [review]

**Pull request merged**